ACE/ace/OS_NS_sys_socket.inl

```diff
@@ -136,9 +137,7 @@ ACE_OS::closesocket (ACE_HANDLE handle)
 
   ACE_SOCKCALL_RETURN (::closesocket ((SOCKET) handle), int, -1);
 #else
-  //FUZZ: disable check_for_lack_ACE_OS
-  return ::close (handle);
-  //FUZZ: enable check_for_lack_ACE_OS
+  return ACE_OS::close (handle);
```

Comment:
Do we know why the old code went directly to `::close` instead of `ACE_OS::close`?

Reply:
I'm not sure. Maybe by mistake and it hasn't caught a platform with `ACE_LACKS_CLOSE`?